lib/internal/Magento/Framework/Filter/Test/Unit/LocalizedToNormalizedTest.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Copyright 2025 Adobe
+ * All Rights Reserved.
+ */
+declare(strict_types=1);
+
+namespace Magento\Framework\Filter\Test\Unit;
+
+use Magento\Framework\Filter\LocalizedToNormalized;
+use PHPUnit\Framework\TestCase;
+use NumberFormatter;
+
+class LocalizedToNormalizedTest extends TestCase
+{
+
+    /**
+     * @param string $value
+     * @param array $options
+     * @param string|array $expectedValues
+     *
+     * @dataProvider localizedToNormalizedDataProvider
+     */
+    public function testLocalizedToNormalized($value, $options, $expectedValue)
```

Comment:
Please add parameter type declarations

for e.g.
```suggestion
    public function testLocalizedToNormalized(string $value, $options, $expectedValue)
```